src/main/java/org/variantsync/diffdetective/variation/diff/DiffNode.java

```diff
@@ -705,23 +723,28 @@ public String toTextDiffLine() {
      * This will return the diff of the entire subgraph starting with this node as root.
      * Recursively invokes {@link DiffNode#toTextDiffLine()} on this node and all its descendants.
      * @return The diff from which this subgraph was parsed, reconstructed as accurately as possible.
+     *
+     * @deprecated because buggy
      */
-    public String toTextDiff() {
+    @Deprecated
```

Comment:
No, the produced patch is just wrong in case of moved subtrees. `toTextDiff` traverses the DiffTree as if it would be tree which it is not. It thus prints shared subtrees twice (e.g., an unchanged node below one added and one deleted node). It's probably better to just delete this method.

Reply:
deleted